src/index.html

```diff
@@ -5,6 +5,168 @@
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
+    <section class="home-view">
+      <h1>punny name here</h1>
+      <nav class="nav-bar">
```

Comment:
Let's put the nav element outside the home-view section.

Reply:
Let's add a <header> tag before our <main> tag and include our <nav> section inside of that <header> tag.